src/foam/u2/view/ChoiceView.js

```diff
@@ -245,7 +248,9 @@ foam.CLASS({
             .end();
         }
 
-        return self.E().translate(self.text + ".name", self.text);
+        return self.E().add(self.text$.map(t => {
+          return t ? self.translationService.getTranslation(foam.locale, t + '.name', t) : '';
```

Comment:
You've imported translactionService as optional with the ?, which is correct, but then you need to check that it's available. So return t && translationService ? ...